src/components/list/_list.scss

```
@@ -63,6 +63,11 @@
     &__prefix,
     &__suffix {
         font-family: $font-mono;
+        .ons-icon {
+            &--check-green {
```

```suggestion
        .ons-icon--check-green {
```

This may as well be this because its not adding anything by splitting them out